specification/synapse/data-plane/Microsoft.Synapse/preview/2021-07-01-preview/symsSync.json

```diff
@@ -0,0 +1,268 @@
+            "schema": {
+              "$ref": "#/definitions/deleteObject"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Request Accepted Response.",
+            "schema": {
+              "$ref": "#/definitions/acceptedResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/workspace.json#/definitions/ErrorContract"
+            }
+          }
+        }
+      }
+    },
+    "/symssync/getrequeststatus": {
+      "get": {
+        "tags": [
+          "symssync"
+        ],
+        "operationId": "SymsSync_GetRequestStatus",
+        "description": "Gets status of a request",
+        "x-ms-examples": {
+          "Register files in Syms": {
+            "$ref": "./examples/SymsSync_GetRequestStatus.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request Success Response.",
+            "schema": {
+              "$ref": "#/definitions/successResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/workspace.json#/definitions/ErrorContract"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "acceptedResponse": {
+      "type": "object",
+      "properties": {
+        "requestId": {
+          "type": "string"
+        }
+      }
+    },
+    "successResponse": {
+      "type": "object",
+      "properties": {
+        "requestStatus": {
+          "type": "string"
+        }
+      }
+    },
+    "registerObject": {
+      "type": "object",
+      "required": [
+        "inputFolder",
+        "databaseName"
+      ],
+      "properties": {
+        "inputFolder": {
+          "type": "string",
+          "example": "xyz"
+        },
+        "databaseName": {
+          "type": "string",
+          "example": "xyz"
+        }
+      }
+    },
+    "updateObject": {
+      "type": "object",
+      "required": [
+        "inputFolder",
+        "databaseName"
+      ],
+      "properties": {
+        "inputFolder": {
+          "type": "string",
+          "example": "xyz"
+        },
+        "databaseName": {
+          "type": "string",
+          "example": "xyz"
+        }
+      }
+    },
+    "deleteObject": {
+      "type": "object",
+      "required": [
+        "databaseName"
+      ],
+      "properties": {
+        "databaseName": {
+          "type": "string",
+          "example": "xyz"
+        }
+      }
+    }
```

Comment:
Can we add some descriptions for these models? In the SDK, the descriptions will be used as document comments. It will help SDK users to understande them.

You can reference [https://github.com/Azure/azure-rest-api-specs/blob/master/specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/library.json#L444](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/library.json#L444)